### DISCUSSION TITLE
Hello, GitHub Community! 😀

### DISCUSSION THREAD

**Discussion by BLCNYY** [community/community#56086] [GitHub Education]
### Hi everyone,

<h1>Hello!</h1>
<p>I'm <b>BLCNYY</b>, I'm a <b>15-years-old Turkish guy</b> who is <b>interested in tech</b> since he started to involve into <b>internet</b>. I'm <b>active on Twitter since <i>October 2014</i></b>, <i>(considering that I have learned how to read/write in September 2014)</i>. I'm started to interest in arithmetic math about 3 years ago. In <i>Summer 2022</i>, I have learned Web Development basics (HTML & CSS). In <i>Early 2023</i>, I have started to learn Phyton.<br><br>My goal is to participate in a big team or mostly create one myself where all developers togethered to create a <b>better future for tech</b>, especially for <b>AI</b>. I'm ready to overcome every difficulty in front of me to become a developer on my way!</p>

**Comment by neondev27** [comment]
Go man!

**Comment by NoelTejeda** [comment]
Felicidades Bro, sigue adelante en tu camino, superándote cada día

**Comment by samus-aran** [comment]
👋 Welcome to the GitHub Community on Discussions, @BLCNYY  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
